### DISCUSSION TITLE
warning Checkstyle: Falta el comentario Javadoc.

### DISCUSSION THREAD

**Discussion by alesiosaba** [dds-jv/foro#213] [Consultas de herramientas] [answered]
El IDE me marca este warning en todos los archivos .java

Por ejemplo en la clase Venta:

![image](https://user-images.githubusercontent.com/62452207/116570472-adfb4b80-a8e0-11eb-92c8-be6670492285.png)

En la clase Prenda:

![image](https://user-images.githubusercontent.com/62452207/116570659-d2572800-a8e0-11eb-9b4f-69ded9667403.png)

¿Es un warning importante? ¿Debería resolverlo para la entrega del TP?

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola! No deberías tener ese warning. ¿Puede ser que hayas configurado checkstyle con la configuración de Sun en lugar de la de Google?

**Reply by alesiosaba** [reply]
Tengo checkstyle configurado con "Google Checks"

![image](https://user-images.githubusercontent.com/62452207/116592055-ec4f3580-a8f5-11eb-9109-f62e260ba4e6.png)

**Reply by flbulgarelli** [reply]
OK, estuvimos investigando y si cambias a la versión 8.35 no debería saltarte ese warning. Decinos si con eso se va. 

En cualquier caso: 
 
  1. No, no es un warning importante, o al menos no vamos a darle importancia en la materia. 
  2. En teoría ese warning tampoco debería saltarte si corrés `mvn verfy`, que es lo que realmente vale para la entrega de los trabajos prácticos individuales. Si de todas formas seguís teniendo ese warning via la consola, ¡avisanos!

**Reply by alesiosaba** [reply]
Al correr ```mvn verify``` solo recibo el siguiente warning por consola:

![image](https://user-images.githubusercontent.com/62452207/116823336-49e3bc00-ab5a-11eb-9fb6-d4a06a8bd524.png)

Me parece raro porque tengo configurada la versión 8.41.1

El warning *Checkstyle: Falta el comentario Javadoc* desapareció al abrir nuevamente el IDE después de varios días. No había cambiado ni actualizado la versión de checkstyle todavía.

**Reply by nanderson87** [reply]
Hola @alesiosaba:

Eso tiene una explicación y es porque finalmente son checkstyle diferentes. Uno el que configuraste en el IDE que seguramente es la versión 8.41.1 y otro que es el que usa maven. Maven para correr los comandos utiliza el archivo de configuración del `pom.xml`. Por lo que pueden ser dos checkstyle diferentes.

En esto caso podes omitir el warning, porque es solo algo que nos esta avisando que se puede usar una versión más actual en maven.
